bin/vcf2gvf.py

```diff
@@ -481,6 +481,7 @@ def add_functions(gvf, annotation_file, clade_file, strain):
         if args.strain.startswith(strain):
             cladefile_strain = strain
 
+
```

Comment:
Parsing the corresponding lineages and other attributes from the clade file. 

- Maybe look at the `available_strains` list that contains all possible lineages and map it to other columns using the index